arangod/Aql/CalculationNodeVarFinder.h

```diff
@@ -30,7 +30,7 @@
 namespace arangodb {
 namespace aql {
 
-class CalculationNodeVarFinder final : public WalkerWorker<ExecutionNode> {
+class CalculationNodeVarFinder final : public WalkerWorker<ExecutionNode, false> {
```

Comment:
Although `true` / `false` works here, I find this harder to read/understand than necessary.
Can't we use something like `WalkerWorkerType::Unique` and `WalkerWorkerType::NonUnique` here? That would help me a lot looking at code.
That should also help against mindless copying&pasting code in the future.

Reply:
Ah, that's a good idea. I'll change that.